### PR TITLE
added the part for extracting date and events from the text

### PR DESCRIPTION
**Pull request by ansjin**
Added the sample output data here https://github.com/MusicConnectionMachine/api/issues/22

### CODE REVIEW

File: algorithms/date_event_extraction/SocketNER.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+         "-loadClassifier", self.pathToNER + self.classifier,
+         "-port", self.port, "-outputFormat", "inlineXML"
+      ]
+   )
+   // I don't know why server's stderr stream gets
+   // all output and why stdout don't.
+   self.server.stderr.on("data", reader)
+//  console.log(self.server);
+   // Server would finish loading, when it flushes
+   // out 'done [x secs]'
```
Comment (RBirkeland) [inline algorithms/date_event_extraction/SocketNER.js:28]: Should be cleaned up

**Review by RBirkeland — CHANGES_REQUESTED** [review]

File: algorithms/date_event_extraction/SocketNER.js (hunk 1 above)
Comment (ansjin) [reply algorithms/date_event_extraction/SocketNER.js:28]: can you explain a bit more what should be cleaned up ? are you referring to comments ?

Comment (RBirkeland) [reply algorithms/date_event_extraction/SocketNER.js:28]: I mean the comment about the console/logger error, I noticed now it was not included in the review preview :) If stderr works, lets just use that?

File: algorithms/date_event_extraction/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "ISC",
+  "private": true,
+  "dependencies": {
+    "body-parser": "~1.0.1",
+    "child_process": "^1.0.2",
+    "clone": "^2.1.1",
+    "connect": "^3.2.0",
+    "deasync": "^0.1.9",
+    "express": "^4.0.0",
+    "fs": "0.0.1-security",
```
Comment (Henni) [inline algorithms/date_event_extraction/package.json]: this shouldn't be in here (as discussed here https://github.com/MusicConnectionMachine/RelationshipsG3/pull/24#discussion_r108437433)

**Review by Henni — CHANGES_REQUESTED** [review]

**Review by Henni — APPROVED** [review]

File: algorithms/date_event_extraction/SocketNER.js (hunk 1 above)
Comment (ansjin) [reply algorithms/date_event_extraction/SocketNER.js:28]: I don't know, In stdout it was not showing up. I just took this part from the stanford NLP wrapper.

**Review by RBirkeland — APPROVED** [review]

**Pull request merged**